### PR TITLE
Jfg 902.lombok

### PR DESCRIPTION
**Pull request by SokolAndrey**
Lombok in action 👍

### CODE REVIEW

File: chassis/core/pom.xml [hunk 1]
```diff
@@ -276,5 +277,16 @@
...
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.16.10</version>
```
Comment (anton-antonenko) [inline chassis/core/pom.xml]: Don't forget to extract all versions)

Comment (a-badaev) [inline chassis/core/pom.xml]: move version to parent pom's dependency managment

File: chassis/core/pom.xml [hunk 2]
```diff
@@ -276,5 +277,16 @@
             <artifactId>jackson-core</artifactId>
             <version>${jackson.version}</version>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
```
Comment (a-badaev) [inline chassis/core/pom.xml]: move version to parent pom's dependency managment

File: chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+import java.util.List;
+
+/**
+ * @author asokol
+ *         created 10/18/16
+ */
+@Builder
+@ToString
+public class JTestConfiguration {
```
Comment (anton-antonenko) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestConfiguration.java]: What about getters/setters and equals/hashCode ? Don't we need them?

File: chassis/core/src/test/java/com/griddynamics/jagger/user/test/configurations/JTestConfigurationTest.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+                .build();
+
+        testConfiguration = JTestConfiguration.builder()
+                .testGroups(Collections.singletonList(testGroup))
+                .build();
+
+
+    }
+
+    @Test
```
Comment (anton-antonenko) [inline chassis/core/src/test/java/com/griddynamics/jagger/user/test/configurations/JTestConfigurationTest.java]: This test seems to be always successful. Why don't you check result with expected value?

File: chassis/core/src/test/java/com/griddynamics/jagger/user/test/configurations/JTestConfigurationTest.java [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+import static org.junit.Assert.*;
+
+/**
+ * @author asokol
```
Comment (a-badaev) [inline chassis/core/src/test/java/com/griddynamics/jagger/user/test/configurations/JTestConfigurationTest.java]: Should be description here

File: chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/load/JRpsLoad.java [hunk 6]
```diff
@@ -0,0 +1,14 @@
+package com.griddynamics.jagger.user.test.configurations.load;
+
+import lombok.Builder;
+import lombok.ToString;
+
+@Builder
```
Comment (a-badaev) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/load/JRpsLoad.java]: Should be description here

File: chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestGroup.java [hunk 7]
```diff
@@ -0,0 +1,16 @@
+package com.griddynamics.jagger.user.test.configurations;
+
+import lombok.Builder;
+import lombok.ToString;
+
+import java.util.List;
+
+/**
+ * @author asokol
```
Comment (a-badaev) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestGroup.java]: Should be description here

File: chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestConfiguration.java [hunk 8]
```diff
@@ -0,0 +1,18 @@
+package com.griddynamics.jagger.user.test.configurations;
+
+import lombok.Builder;
+import lombok.ToString;
+
+import java.util.List;
+
+/**
+ * @author asokol
```
Comment (a-badaev) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestConfiguration.java]: Should be description here

File: chassis/core/src/main/java/com/griddynamics/jagger/user/TestDescription.java [hunk 9]
```diff
@@ -20,8 +21,8 @@
 
     private List<KernelSideObjectProvider<Validator>> validators;
 
-    private List<KernelSideObjectProvider<ScenarioCollector<Object,Object,Object>>> metrics;
-    private List<KernelSideObjectProvider<ScenarioCollector<Object,Object,Object>>> standardCollectors;
+    private List<KernelSideObjectProvider<ScenarioCollector<Object, Object, Object>>> metrics;
```
Comment (a-badaev) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/TestDescription.java]: It is not Generics style

**Comment by SokolAndrey** [comment]
:(

**Pull request closed without merging**